packages/discovery-provider/src/models/users/usdc_transactions_history.py

```diff
@@ -9,6 +9,9 @@
 class USDCTransactionType(str, enum.Enum):
     purchase_content = "PURCHASE_CONTENT"
     transfer = "TRANSFER"
+    prepare_withdrawal = "PREPARE_WITHDRAWAL"
+    recovery = "RECOVERY"
```

Comment:
nit: It _might_ make sense to name this "RECOVER_WITHDRAWAL" or something more specific to withdrawals

Reply:
I originally had it like that. But we initiate recovery arbitrarily on startup and loading of the pay and earn page. So it wasn't exactly a 1:1 correspondence. I thought it would make more sense from a ledger perspective in cases where we recover some funds that _weren't_ from a withdrawal. Though I know that's an edge case at the moment.

Reply:
Updated this to be "Recover Withdrawal". Will need to update the client memo as well, but since this is currently the only use case for staging tokens in the root wallet, I'm fine with it.